components/backup/base/kustomization.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Let's put namespace here so we do not have to repeat it in every resource we will have to create in this component
```suggestion
kind: Kustomization
namespace: openshift-adp
```